drivers/sensor/max17055/max17055.c

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/i2c.h>
+#include <drivers/sensor.h>
+
+#include "max17055.h"
+
+#define DT_DRV_COMPAT maxim_max17055
+
+static int max17055_reg_read(struct max17055_data *max17055, int reg_addr,
+			     s16_t *val)
+{
+	u8_t i2c_data[2];
+	int status;
+
+	status = i2c_burst_read(max17055->i2c, DT_INST_REG_ADDR(0), reg_addr,
+				i2c_data, 2);
+	if (status < 0) {
+		LOG_ERR("Unable to read register");
+		return -EIO;
+	}
+
+	*val = i2c_data[1] << 8 | i2c_data[0];
```

Comment:
Precedence is understood, but better to add braces for clarity.. 
```
*val = (i2c_data[1] << 8) | i2c_data[0];
```

Reply:
done